extensions/jms/src/main/java/org/mule/extensions/jms/api/source/JmsListener.java

```diff
@@ -116,26 +112,36 @@
 
   @Parameter
   @Optional
+  @Example(EXAMPLE_CONTENT_TYPE)
   @Summary("The content type of the message body")
   private String contentType;
 
   @Parameter
   @Optional
+  @Example(EXAMPLE_ENCODING)
   @Summary("The encoding of the message body")
   private String encoding;
 
+  @Parameter
+  @Optional(defaultValue = "true")
```

Comment:
sync is less performant but more predictable and maintains 3.x-like behaviour. The user can always enable async if they don't care about mixing messages ack's